### ISSUE TITLE
AVT 3 - Carbon X Tile Component VO does not announce state of expandable tiles

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#1979]
## Environment
macOS Mojave version 10.14.2
Chrome version 72.0.3626.96 (Official Build) (64-bit)

## Detailed description
Control-Option-Right Arrow to the Selectable Tile. Press the Space key to select the tile. 
Expected Result: VoiceOver announces the status as checked.
Actual result: VoiceOver does not announce the status. 
Note: Add aria-checked="true" when the tile is checked and aria-checked=false" when the tile is unchecked. 

Control-Option-Right Arrow to the Expandable Tile. Press the Space key to select the tile. 
Expected Result: VoiceOver announces the status as expanded.
Actual result: VoiceOver does not announce the status. 
Note: Add aria-expandeded="true" when the tile is expandeded and aria-expanded=false" when the tile is closed.

**Comment by dakahn** [comment]
Great catch! Thanks @snidersd

**Comment by snidersd** [comment]
Retested and the state is not announced for the expandable tile. The selectable tile state is announced correctly.

**Comment by abbeyhrt** [comment]
This was closed via #4055
